ci/scripts/check_dv_sw_images.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Checks for Bazel targets in sim DV cfg files.
+"""
+
+import sys
+import hjson
+from pathlib import Path
+
+from lib.bazel_query import BazelQueryRunner
+
+# List of fake targets added to please the linker:
+FAKE_TARGETS = [
+    "@manufacturer_test_hooks//:example_test",
+]
+
+
+def check_sw_image(name, sw_image, valid_sw_images):
+    # The format is target:type[:flags]...
```

Comment:
Nit: Probably worth saying "The format of sw_image is..." (it took me a couple of seconds to figure out what was going on with this comment).

The extra colon is also a bit confusing: would it make sense to give an example value in the comment?

Reply:
Sure, I'll update the comment